spec/spec_helper.rb

```diff
@@ -1,5 +1,6 @@
 require "bundler/setup"
 require "opera"
+require "pry"
```

Comment:
Was this intended? I'm OK having it if it was intended but the majority of specs runs are automated and would not use pry. Not sure how much overhead it adds to the pipeline though.